app/src/test/java/org/simple/clinic/facility/change/FacilityChangeScreenControllerTest.kt

```diff
@@ -331,7 +337,51 @@ class FacilityChangeScreenControllerTest {
 
   @Suppress("unused")
   fun `params for location updates`() = listOf(
-      LocationUpdate.Unavailable,
-      LocationUpdate.Available(location = Coordinates(0.0, 0.0), timeSinceBootWhenRecorded = Duration.ofNanos(0))
+      Unavailable,
+      Available(location = Coordinates(0.0, 0.0), timeSinceBootWhenRecorded = Duration.ofNanos(0))
   )
+
+  @Test
+  fun `when both facilities and location are fetched only then should facilities be shown`() {
```

Comment:
Does this mean if location is not fetched then facilities should not be shown?

Reply:
Yes

Reply:
The spec can be found in the description box on zeplin: https://zpl.io/2vWEqR5

Reply:
The description on Zeplin says: 
> If location detection fails within 5 seconds, this state will be resolved and the user would see the list of facilities
This test name seems to contradict that or maybe I don't understand what do you mean by "location is fetched"
Does it mean `LocationUpdate` is received?

Reply:
> Does it mean `LocationUpdate` is received?

yes! 

This line is perhaps more relevant from the spec:

> The loading screen is resolved when 1) the list of facilities has been fetched AND 2) the rough location of the user has been detected.